src/BugBundle/Entity/Issue.php

```diff
@@ -0,0 +1,616 @@
+<?php
+
+namespace BugBundle\Entity;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * Issue
+ *
+ * @ORM\Table()
+ * @ORM\Entity(repositoryClass="BugBundle\Entity\IssueRepository")
+ * @ORM\HasLifecycleCallbacks()
+ */
+class Issue
+{
+    const TYPE_BUG = 1;
+    const TYPE_SUBTASK = 2;
+    const TYPE_TASK = 3;
+    const TYPE_STORY = 4;
+
+
+    /**
+     * /**
+     * @var integer
+     *
+     * @ORM\Column(name="id", type="integer")
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+
+    /**
+     * @var string
+     * @Assert\NotBlank()
+     * @Assert\Length(min=5)
+     * @ORM\Column(name="summary", type="string", length=1000)
+     */
+    private $summary;
+
+    /**
+     * @var string
+     * @Assert\Length(min=3)
+     * @Assert\Length(max=3)
+     * @ORM\Column(name="code", type="string", length=10)
+     */
+    private $code;
+
+    /**
+     * @var string
+     * @Assert\NotBlank()
+     * @Assert\Length(min=7)
+     * @ORM\Column(name="description", type="string", length=10000)
```

Comment:
type="string", length=10000

so, you have assert for `@Assert\Length(min=7)` but don't add any assert for max? `length=10000`?